cmd/permission/main.go

```diff
@@ -2,24 +2,23 @@ package main
 
 import (
 	"fmt"
+
+	"github.com/openslides/openslides-permission-service/internal/permission"
 )
 
 type fakeDataProvider struct{}
 
-type Fqfield = string
-type Value = interface{}
-
-func (dp *fakeDataProvider) get(fqfields []Fqfield) map[Fqfield]Value {
-	m := make(map[Fqfield]Value)
+func (dp *fakeDataProvider) Get(fqfields []permission.Fqfield) map[permission.Fqfield]permission.Value {
+	m := make(map[permission.Fqfield]permission.Value)
 	for i, val := range fqfields {
 		m[val] = i
 	}
 	return m
 }
 
 func main() {
-	myDataProvider := fakeDataProvider{}
-	myFakeData := []Fqfield{"Foo", "Bar"}
-	result := DoIt(myDataProvider, myFakeData)
+	myDataProvider := &fakeDataProvider{}
+	myFakeData := []permission.Fqfield{"Foo", "Bar"}
+	result := permission.DoIt(myDataProvider, myFakeData)
```

Comment:
The DoIt function is in the permission package. The Package name is part of the function name.